extensions/GoogleAnalytics/template/en/default/hook/global/header-additional_header.html.tmpl

```diff
@@ -0,0 +1,28 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% RETURN UNLESS Bugzilla.params.google_analytics_tracking_id && !user.in_group(Param('insidergroup')) %]
+
+<meta name="google-analytics" content="[% Bugzilla.params.google_analytics_tracking_id FILTER html %]" data-location="
+  [%~ urlbase FILTER html %][% template.name.replace('\.html\.tmpl$', '') FILTER html ~%]
+  " data-title="
+  [%~ IF bug.defined && bug.groups_in.size ~%]
+    [% bug.id FILTER none %] – (private bug)
+  [%~ ELSIF template.name == 'pages/user_activity.html.tmpl' ~%]
+    User Activity Report
+  [%~ ELSIF template.name == 'pages/user_profile.html.tmpl' ~%]
+    User Profile
+  [%~ ELSE ~%]
+    [% title FILTER html %]
+  [%~ END ~%]
+  ">
+<script async src="https://www.google-analytics.com/analytics
+  [%~ IF Bugzilla.params.google_analytics_debug ~%]
+    _debug
+  [%~ END ~%]
```

Comment:
this can be rewritten as `[%~ "_debug" IF Bugzilla.params.google_analytics_debug ~%]